src/components/CourseForm/CourseForm.jsx

```diff
@@ -81,7 +99,19 @@ export default function CreateCourse() {
 			return;
 		}
 
-		dispatch(addCourse(course));
+		edit
+			? dispatch(
+					thunk_updateCourse(
+						{ ...course, duration: +course.duration },
+						localStorage.getItem('token')
```

Comment:
if you take token from local storage, then there's no sense to pass it, you can do it directly in courses thunk, e.g. create variable and then use it

Reply:
Done!